### PR TITLE
Doc: Update for equipment_reports

### PR DESCRIPTION
**Pull request by pbougue**
Related to PR https://github.com/CanalTP/navitia/pull/2795

Removes the doc of `updated_at` as it is usable, but deprecated (wrong format)

### CODE REVIEW

File: documentation/slate/source/includes/apis.md [hunk 1]
```diff
@@ -2161,7 +2161,7 @@ Equipment_Reports
 ---------------------------------------------
 ``` shell
 #request
-$ curl 'http://api.navitia.io/v1/coverage/sandbox/equipment_reports' -H 'Authorization: 3b036afe-0110-4202-b9ed-99718476c2e0'
+$ curl 'http://api.navitia.io/v1/coverage/<my_coverage>/equipment_reports'
```
Comment (pbougue) [inline documentation/slate/source/includes/apis.md:2162]: Changed it because we have no response on sandbox for that URL.

**Review by pbougue — COMMENTED** [review]
@nonifier some changes may be a matter of taste, feel free to ask to remove some, you're the :gb:-guy :wink:

File: documentation/slate/source/includes/objects.md [hunk 2]
```diff
@@ -658,15 +658,15 @@ A list of object that maps each lines with its associated stop area equipments r
     ...
 ]
 ```
-A list of objects that maps equipments details for each stop area.
+A list of objects that maps equipment details for each stop area.
```
Comment (mbounabi) [inline documentation/slate/source/includes/objects.md]: I do not agree with this change. There might be several equipments in a stop area
```suggestion
A list of objects that maps equipments details for each stop area.
```

File: documentation/slate/source/includes/objects.md [hunk 3]
```diff
@@ -658,15 +658,15 @@ A list of object that maps each lines with its associated stop area equipments r
...
 
 |Field|Type|Description|
 |-----|----|-----------|
-|equipment_details|[equipment-details](#equipment-details)|The equipment details associated with the stop area|
-|stop_area|[stop-area](#stop-area)|The stop area to which the `equipments_details` is associated |
+|equipment_details|[Equipment details](#equipment-details)|The equipment details associated with the stop area|
+|stop_area|[Stop Area](#stop-area)|The stop area to which the `equipment_details` are associated |
```
Comment (mbounabi) [inline documentation/slate/source/includes/objects.md]: `equipment_details` is a structure, a dict, an element but why make it plural?
```suggestion
|stop_area|[Stop Area](#stop-area)|The stop area to which the `equipment_details` is associated |
```

**Review by mbounabi — APPROVED** [review]

File: documentation/slate/source/includes/apis.md [hunk 4]
```diff
@@ -2208,9 +2208,10 @@ HTTP/1.1 200 OK
 Also known as the `"/equipment_reports"` service.
 
 This service provides the state of equipments such as lifts or elevators that are giving you better accessibility to public transport facilities.
-The endpoint will report accessible equipement per stop area and per line. Which means that an equipment detail is reported at the stop area level, with all stop areas gathered on a per line basis.
+The endpoint will report accessible equipment per stop area and per line. Which means that an equipment detail is reported at the stop area level, with all stop areas gathered per line.
 Some of the fields (cause, effect, periods etc...) are only displayed if a realtime equipment provider is setup with available data. Otherwise, only information provided by the NTFS will be reported.
-For more information, refer to [equipment-reports](#equipment-reports) API description.
+For more information, refer to [Equipment reports](#equipment-reports) API description.
+You can access it via that kind of url: <https://api.navitia.io/v1/{a_path_to_a_resource}/equipment_reports>
```
Comment (nonifier) [inline documentation/slate/source/includes/apis.md]: very familiar 
```suggestion
Can be accessed via: <https://api.navitia.io/v1/{a_path_to_a_resource}/equipment_reports>
```

File: documentation/slate/source/includes/objects.md [hunk 5]
```diff
@@ -627,7 +627,7 @@ Cities are mainly on the 8 level, dependant on the country
 (<http://wiki.openstreetmap.org/wiki/Tag:boundary%3Dadministrative>)
 
 
-### Equipment-reports
+### Equipment reports
```
Comment (nonifier) [inline documentation/slate/source/includes/objects.md]: Is this not breaking anchor links ?!

File: documentation/slate/source/includes/objects.md [hunk 6]
```diff
@@ -639,15 +639,15 @@ Cities are mainly on the 8 level, dependant on the country
...
 
 |Field|Type|Description|
 |-----|----|-----------|
 |line|[line](#line)|The line to which equipments are associated |
-|stop_area_equipments|[stop-area-equipments](#stop-area-equipments)|A list of objects that describes equipments for each stop area |
+|stop_area_equipments|[stop area equipments](#stop-area-equipments)|A list of objects that describes equipments for each stop area |
 
 
-### Stop-area-equipments
+### Stop area equipments
```
Comment (nonifier) [inline documentation/slate/source/includes/objects.md]: Same here, any broken anchor links ?

File: documentation/slate/source/includes/objects.md (hunk 2 above)
Comment (nonifier) [reply documentation/slate/source/includes/objects.md]: I tend to agree with mbounabi.

File: documentation/slate/source/includes/objects.md [hunk 7]
```diff
@@ -843,7 +841,7 @@ Special Parameters
 
 ### datetime
 
-A date time with the format YYYYMMDDThhmmss
+A date time with the format YYYYMMDDThhmmss, considered local to the coverage used.
```
Comment (nonifier) [inline documentation/slate/source/includes/objects.md]:
```suggestion
A date time with the format YYYYMMDDThhmmss, considered local to the coverage being used.
```

**Review by nonifier — APPROVED** [review]

**Pull request merged**